packages/terra-button/src/_variables.scss

```diff
@@ -15,8 +15,7 @@ $terra-button-neutral-color-scheme: (
   box-shadow: var(--terra-button-box-shadow-neutral, none),
   color: var(--terra-button-color-neutral, #1c1f21),
   focus-background-color: var(--terra-button-focus-background-color-neutral, #dedfe0),
-  focus-border-color: var(--terra-button-focus-border-color-neutral, #c8cacb),
-  focus-box-shadow: var(--terra-button-focus-box-shadow-neutral, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35)),
```

Comment:
Terra-button screenshots are updated incorrectly due to removal of these styles. : https://github.com/cerner/terra-core/pull/3998/files#diff-6629a708ca1f301df6d09a8ba8c94580d13450beab7f124cc12e923ee63048ee 
Focus styles are not visible on focused button !!

Reply:
Updated.